tests/Database/DatabaseEloquentModelTest.php

```diff
@@ -1152,6 +1152,40 @@ public function testModelAttributesAreCastedWhenPresentInCastsArray()
 	}
 
 
+	public function testModelAttributeCastingPreservesNull()
```

Comment:
Maybe, but this is basically what's throughout the whole codebase.

Unless we're doing a sweeping change everywhere else, there's no need for this to be so different.